.circleci/config.yml

```diff
@@ -15,18 +15,64 @@ jobs:
           command: git submodule update --init
       - run: apt update
       - run: apt install -y zip
-      - run: make release
+      - run: make prepare
       - run: make
+      - run: make clean
+      - run: make release
       - run:
-          name: make driver
+          name: build stub release drivers
           command: |
-              cd release/sht3x && make && make clean && cd -
-              cd release/shtc1 && make && make clean && cd -
+              cd release/sht3x
+              make CONFIG_I2C_TYPE=hw_i2c
+              make CONFIG_I2C_TYPE=sw_i2c
+              make clean
+              cd -
+              cd release/shtc1
+              make CONFIG_I2C_TYPE=hw_i2c
+              make CONFIG_I2C_TYPE=sw_i2c
+              make clean
+              cd -
       - run:
-          name: Move zip files to artifacts directory
+          name: build Linux release drivers
           command: |
-            mkdir -p /releases
-            mv release/*.zip /releases/
+              cd release/sht3x
+              make CONFIG_I2C_TYPE=hw_i2c hw_i2c_impl_src=hw_i2c/sample-implementations/linux_user_space/sensirion_hw_i2c_implementation.c
+              make CONFIG_I2C_TYPE=sw_i2c sw_i2c_impl_src=sw_i2c/sample-implementations/linux_user_space/sensirion_sw_i2c_implementation.c
+              make clean
+              cd -
+              cd release/shtc1
+              make CONFIG_I2C_TYPE=hw_i2c hw_i2c_impl_src=hw_i2c/sample-implementations/linux_user_space/sensirion_hw_i2c_implementation.c
+              make CONFIG_I2C_TYPE=sw_i2c sw_i2c_impl_src=sw_i2c/sample-implementations/linux_user_space/sensirion_sw_i2c_implementation.c
+              make clean
+              cd -
 
-      - store_artifacts:
+  deploy:
+    docker:
+      - image: gcc:5.5
+    steps:
+      - checkout
+      - run:
+          name: update common repo
+          command: git submodule update --init
+      - run: apt update
+      - run: apt install -y zip
+      - run: make release
```

Comment:
`Move zip files to artifacts directory` is missing (so the artifacts are empty).

Reply:
Done